### PR TITLE
Modified information removed ul's to conform to the site's css

### PR DESCRIPTION
**Pull request by deidyomega**
The site's CSS removes UL's.  I imagine for nav.  I've removed them from my .md as it doesn't look good currently.

### CODE REVIEW

**Comment by abettermap** [comment]
Hmmm `<ul>` would be nice. I'll take a look this weekend when I'm not on a phone. If it's not an easy fix I'll just merge your PR.

**Comment by deidyomega** [comment]
If I find and fix it, would you accept my PR?

**Comment by abettermap** [comment]
Yep. Sorry if slow turnaround for PR's though, 9-to-5 usually means weekends-only for volunteer stuff.

**Comment by abettermap** [comment]
Plenty of other contributors with same permissions though, so maybe someone will beat me to it. If not... weekend.

File: _posts/registry/2017-07-22-matthew-harris.md [hunk 1]
```diff
@@ -10,18 +10,10 @@ categories: registry
 Hi, my name is Matthew Harris.  I am the CTO of a SaaS company in the finance sector.  I'd be happy consult or work on any project.
```
Comment (abettermap) [inline _posts/registry/2017-07-22-matthew-harris.md]: Missed a `to` in the second sentence. 🙀

**Comment by abettermap** [comment]
Ok I opened an issue for the list problem: #103 

Going to look into it shortly...

File: _posts/registry/2017-07-22-matthew-harris.md (hunk 1 above)
Comment (deidyomega) [inline _posts/registry/2017-07-22-matthew-harris.md]: Whoops, yea I suck a proof reading.  I'll do another PR.

**Comment by abettermap** [comment]
For some reason I can't get your post to show up at all. I've tried changing the content, duplicating the file with a different name, and it still doesn't show up. I was able to edit others' posts and duplicate them as a test with no problem, but yours doesn't show up. 

It's not a caching thing as I restarted Jekyll. Also tried checking out the file at a previous commit. Also made it sure the template doesn't have some kind of item limit. 

It's working for you I take it (except for #102), so any ideas?

**Comment by JordanMajd** [comment]
The post is not displaying because the date is in the future. 
```
date:  2017-07-22 18:30
```

Should be:

```
date:  2017-06-22 18:30
```

The site's `_config.yml`  contains a variable called `future`, that when set to `true` allows posts with a date in the future to be published. I think by default this variable is false. More info in the [Jekyll configdocs](https://jekyllrb.com/docs/configuration/#build-command-options).

**Comment by deidyomega** [comment]
Ok!  I've fixed the date, and went back to the version prior to the removal of the \<ul\>'s

**Comment by abettermap** [comment]
Good to know about the `future` thing, that was a bugger to troubleshoot.

**Pull request merged**